packages/dev/core/src/Inputs/scene.inputManager.ts

```diff
@@ -97,7 +97,7 @@ export class InputManager {
     private _previousButtonPressed: number;
     private _currentPickResult: Nullable<PickingInfo> = null;
     private _previousPickResult: Nullable<PickingInfo> = null;
-    private _totalPointersPressed = 0;
+    private _activeTouchIds: number[] = [];

```

Comment:
```suggestion
    private readonly _activeTouchIds: number[] = [];

```

Reply:
Seems like the code would be slightly simpler with a `Set<number>` as well?

Reply:
I did some testing, and deleting from a set will deallocate memory and could trigger more frequent garbage collection, as would the splice I originally had in this change.  Even though this isn't a super high frequency code path, I decided to avoid the GC pressure and go with something similar to how we manage this in the webDeviceInputSystem class - with a slot-based array that doesn't every shrink.  Not only can that run about twice as many operations per ms in my testing, but it accrues no GC pressure.  Upstream of this, we limit the total number of concurrent pointer operations to the maxTouchPoints supported by the browser, so I've gone with a grow-as-you-need-to approach here.  As pointers are lifted up, slots will open up in the array that we'll reuse, so in practice, it will never grow beyond what the upstream deviceInputSystem allows (5 in practice from what I see).